CHANGES.md

```diff
@@ -1,5 +1,124 @@
+optionally allows Linkerd to produce Apache-style request logs, optional
+support for `iptables-nft`, and a host of other improvements and performance
+enhancements.
+
+**Upgrade notes**: Please see the [upgrade instructions](upgrade-2120). <!-- TODO fix link -->
+
+* Proxy
+  * Added a `config.linkerd.io/shutdown-grace-period` annotation to configure
+   the proxy's maximum grace period for graceful shutdown
+  * Added a new `iptables-nft` mode for `proxy-init`
+  * Fixed an issue where the proxy-injector would break when using
+    `nodeAffinity` values for the control plane
+
+* Control Plane
+  * Added support for per-route policy by supporting AuthorizationPolicy
+    resources which target HttpRoute resources
+  * Removed kube-system exclusions from watchers to fix service discovery
+    for workloads in the kube-system namespace (thanks @JacobHenner)
+  * Updated healthcheck to ignore `Terminated` state for pods (thanks
+    @AgrimPrasad!)
+  * Updated the default policy controller log level to `info`; the controller
+    will now emit INFO level logs for some of its dependencies
+  * Added probe authorization by default, allowing clusters that use a
+    default `deny` policy to not explicitly need to authorize probes
+  * Fixed an issue where certain control plane components were not restarting
+    as necessary after a trust root rotation
+  * Fixed an issuer where the `--default-inbound-policy` setting was not
+    being respected
+
+* CLI
+  * Fixed the `linkerd check` command crashing when unexpected pods are found
+    in a Linkerd namespace
+  * Updated the `linkerd authz` command to support AuthorizationPolicy and
+    HttpRoute resources
+  * Updated linkerd check to allow RSA signed trust anchors (thanks
+    @danibaeyens)
+  * Fixed some invalid yaml in the viz extension's tap-injector template
+    (thanks @wc-s)
+  * Added support for AuthorizationPolicy and HttpRoute to viz authz command
+  * Added support for AuthorizationPolicy and HttpRoute to viz stat
+  * Added support for policy metadata in linkerd tap
+
+* Helm
+  * Added missing port in the Linkerd viz chart documentation (thanks @haswalt)
+  * Changed the `proxy.await` Helm value so that users can now disable
+    `linkerd-await` on control plane components
+  * Added the `policyController.probeNetworks` Helm value for configuring
+    the networks that probes are expected to be performed from
+
+* Extensions
+  * Added annotations to allow Linkerd extension deployments to be evicted by
+    the autoscaler when necessary
+  * Added ability to run the Linkerd CNI plugin in non-chained (stand-alone) mode
+  * Added a ServiceAccount token Secret to the multicluster extension to support
+    Kubernetes versions >= v1.24
+
+This release includes changes from a massive list of contributors, including
+engineers from Adidas, Intel, Red Hat, Shopify, Sourcegraph, Timescale, and
+others. A special thank-you to everyone who helped make this release possible:
+
```

Comment:
missing:

* evan-hines-firebolt, contributed in [`edge-22.5.2`](https://github.com/linkerd/linkerd2/blob/main/CHANGES.md#edge-2252)
* jtcarnes, contributed in [`edge-22.6.2`](https://github.com/linkerd/linkerd2/blob/main/CHANGES.md#edge-2262)